src/Microsoft.DotNet.Wpf/src/WpfGfx/core/common/utils.cpp

```diff
@@ -129,20 +129,27 @@ UINT GetPaddedByteCount(UINT cbSize)
     // We want precisely one of _X86_, 
     // _AMD64_ or _ARM_ to be defined. 
     // f(A,B,C) =  A'BC' + AB'C' + A'B'C
+/*
 #if !((!defined(_X86_) && defined(_AMD64_) && !defined(_ARM_)) || \
       (defined(_X86_) && !defined(_AMD64_) && !defined(_ARM_)) || \
-      (!defined(_X86_) && !defined(_AMD64_) && defined(_ARM_)))
+      (!defined(_X86_) && !defined(_AMD64_) && defined(_ARM_));
```

Comment:
Was this change intentional? Either we should make the equivalent change or delete this.

Reply:
This macro was commented out.  I've added a case for ARM64 and uncommented it.